lib/__snapshots__/decodeEvent.test.snap

```diff
@@ -15,7 +15,7 @@ Object {
     },
   ],
   "eventFragment": Array [
-    "Transfer(address,address,uint256)",
+    "3",
```

Comment:
does it make sense? i thought it's gonna be a name of the event 🤔

Reply:
I think that JSON.parse losses "Transfer(address,address,uint256)" format.
`console.log(JSON.parse(iface.format(FormatTypes.json)))` prints e.g

```
[
  '3',
  EventFragment {
    name: 'Transfer',
    anonymous: false,
    inputs: [ [ParamType], [ParamType], [ParamType] ],
    type: 'event',
    _isFragment: true
  }
]
```

So I think there's nothing we can do about it in the current impl.

Reply:
that sucks. please leave a comment about this because we loose some information piece and it's clear regression (I saw that on UI)